chat-plugins/info.js

```diff
@@ -36,6 +36,7 @@ exports.commands = {
 		}
 
 		let buf = Chat.html`<strong class="username"><small style="display:none">${targetUser.group}</small>${targetUser.name}</strong> `;
+		if (targetUser.autoconfirmed) buf += ` <small style="color:gray">(ac)</small>`;
```

Comment:
I'd say drivers+ Although we show the effected ac accounts when a punishment goes through so I guess it doesn't matter too much